### PR TITLE
Update Campaigns OKRs for 2020 Q1

### PR DESCRIPTION
**Pull request by mrnugget**

### CODE REVIEW

File: company/okrs/2020_q1.md [hunk 1]
```diff
@@ -117,10 +117,10 @@
 _This OKR is a joint effort between [web](#web) and [core services](#core-servies)._
 
 1. **Campaigns are polished and adopted by customers**
-   1. Document campaigns and write a blog post about it.
-   1. Campaigns are enabled by default and not behind a feature flag.
-   1. Run 3 user tests with Sourcegraph colleagues, collect issues and fix them.
-   1. 3 different customers have together run at least 10 campaigns.
+   1. Document campaigns and write a blog post about it. => Documentation has been greatly improved. No announcement blog post yet, but created multiple demo videos and demo campaigns.
```
Comment (nicksnyder) [inline company/okrs/2020_q1.md:120]: Can you clarify whether announcement blog post is on track to happen in the next month?

Comment (mrnugget) [reply company/okrs/2020_q1.md:120]: It's not planned yet, no. But @christinaforney has a better idea here of how to structure the release.

Comment (mrnugget) [reply company/okrs/2020_q1.md]: I made it clearer in the text that it's not planned yet.

Comment (christinaforney) [reply company/okrs/2020_q1.md]: I think we can target a more informal blog post - perhaps using @unknwon 's real-life campaigns example - or if there is another initiative we could solve inside the Sourcegraph code base using campaigns. It doesn't need to be the formal announcement blog post, just talking about what it is and why it's helping solve a real problem.

**Pull request merged**